web/src/pages/application/EditApplication.js

```diff
@@ -28,7 +30,20 @@ function EditApplication() {
       history.push('/applications'),
     );
   };
-  return <ApplicationEditForm data={application} onCreate={onUpdate} />;
+  return (
+    <Space direction="vertical" style={{ width: '100%' }}>
+      <Collapse>
+        <Panel header="Application Details" key="1">
+          <ApplicationEditForm data={application} onCreate={onUpdate} />
+        </Panel>
+      </Collapse>
+      <Collapse>
+        <Panel header="Tokens" key="2">
+          <GetApplication />
```

Comment:
@Bhumi101097  pass application data as a prop to  `GetApplication` component. And remove getApplication action in  `GetApplication` component.